tests/test_validation/test_validation.py

```diff
@@ -0,0 +1,121 @@
+from rest_framework import status
+from rest_framework.test import APITestCase
+import re
+
+
+class Test_Validation(APITestCase):
```

Comment:
@nyabongoedgar following the PEP8 style guidelines we don't write class names with underscores.

Reply:
Thank you. Class name is now TestValidation(APITestCase).